### ISSUE TITLE
Position calculation problem

### ISSUE THREAD

**Issue by hosh0425** [uzh-rpg/rpg_svo_example#7]
Hi . I am using a monocular camera. I move the camera in each three axes about 2meter but the change in topic /pose/camera is very less than 2meter(about 0.4). Should I scale this data by my self?

**Comment by zhangzichao** [comment]
Yes, for monocular case you do not have the exact scale anyway.  The parameter `map_scale` defines the median  of the scene depth for initialization and the scale is carried on afterwards.

What you can do is to initialize the pipeline by pointing your camera perpendicular to a plane and moving the camera in parallel to it. Then if you know how far the plane is from the camera, say 1 meter, you can set the `map_scale` to 1. This should gives you a roughly correct scale.

**Comment by joshi-bharat** [comment]
I am also having a similar issue while moving clearpath husky around the corridor taking rounds (using monocular camera). **However, the scale decreases when I move the robot around.** Are there any specific parameters I should look at? I have attached some pictures for reference.
![husky_indoor_120_5](https://user-images.githubusercontent.com/17511684/35783369-cc583980-09d3-11e8-9bbe-b3d4f20570f1.png)
![husky_2](https://user-images.githubusercontent.com/17511684/35783371-cdb962fe-09d3-11e8-9932-4dd34bcfce55.png)
